ui/src/app/kvm/views/LXDClusterDetails/LXDClusterDetails.test.tsx

```diff
@@ -42,7 +42,9 @@ describe("LXDClusterDetails", () => {
         </MemoryRouter>
       </Provider>
     );
-    expect(wrapper.find("[data-test='not-found']").exists()).toBe(true);
+    expect(wrapper.find("[data-test='section-header-title']").text()).toBe(
```

Comment:
These tests can probably just test for the existence of `ModelNotFound` with a test id (or check the `modelName` prop for host/cluster) otherwise you're testing the internal string concat that you're already testing in `ModelNotFound.test.tsx`.

Reply:
Fair point, updating now